tuxguitar: Update homepage URL

lhanson opened the pull request:
Update homepage since the current one redirects to spam sites.


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

lhanson committed:
Update homepage URL

The old one redirects to spam pages.

Kek5chen commented:
https://www.tuxguitar.app/ also exists. looks old but it seems up to date. The official github repo only mentions [the com.ar site](https://github.com/helge17/tuxguitar/blob/b95cea5da9662d528ad66f177b40ce709d0d2687/common/TuxGuitar-midi/keyboard-bug/description)

lhanson commented:
> https://www.tuxguitar.app/ also exists. looks old but it seems up to date. The official github repo only mentions [the com.ar site](https://github.com/helge17/tuxguitar/blob/b95cea5da9662d528ad66f177b40ce709d0d2687/common/TuxGuitar-midi/keyboard-bug/description), although as an archived link.

https://www.tuxguitar.app/ doesn't have as much content but at least it references the github repo so I'm tempted to switch to that one. Do you have a preference?

Kek5chen commented:
I think the options all kind of suck because I believe that the homepage should only be set to official homepages. Some "fan-made" sites are cool, but they might go down any time if the author decides that they want to reallocate their server resources. the com.ar seems to be the only one that was at least traceably official. Maybe we just remove it for now so the spam story is out the way? The GitHub repo is pretty informative in itself.

Unless there's a way to find out which site is from the owner of the repo...

lhanson commented:
> I think the options all kind of suck because I believe that the homepage should only be set to official homepages. Some "fan-made" sites are cool, but they might go down any time if the author decides that they want to reallocate their server resources. the com.ar seems to be the only one that was at least traceably official. Maybe we just remove it for now so the spam story is out the way? The GitHub repo is pretty informative in itself.
> 
> Unless there's a way to find out which site is from the owner of the repo...

Should we just use https://github.com/helge17/tuxguitar as the homepage? That's the most official source of info and is less likely to go out of date or disappear.

lhanson committed:
Use github page as homepage URL

There are a couple of unofficial websites (https://tuxguitar.org, https://www.tuxguitar.app), but the Github page seems like the site least likely to disappear or go out of date.
